packages/contact-center/user-state/src/helper.ts

```diff
@@ -1,4 +1,6 @@
 import {useState, useEffect} from "react";
+// TODO: Export & Import this AGENT_STATE_CHANGE constant from SDK
+import { AGENT_STATE_CHANGE } from './constants';
```

Comment:
nit. This import statement doesn't seem to follow the same linting as the previous line?